libs/libc/machine/arm/armv6-m/arch_elf.c

```diff
@@ -216,12 +220,12 @@ int up_relocate(const Elf32_Rel *rel, const Elf32_Sym *sym, uintptr_t addr)
          * lower_insn:
          *
          *  1   1   1   1   1   1
-         *  5   4   3   2   1   0   9  8  7  6  5  4  3  2  1  0 Instructions
+         *  5   4   3   2   1   0  9  8  7  6  5  4   3  2  1  0 Instructions
          * +---+------------------------------------------------+
          * |OP |                                                | 32-Bit
          * +---+--+---+---+---+---------------------------------+
-         * |1   1 |J1 | 1 |J2 |              imm11              | BL
-         * +------+---+---+---+---------------------------------+
+         * |1   1 |J1 | 1 |J2 |            imm11                | BL
+         * +------+---+---+---+--------------------------------+
```

Comment:
GitHub preview shows that `-` is needed here. Please make sure that it is well aligned

Reply:
Done